### DISCUSSION TITLE
Feat：请支持 Cubox

### DISCUSSION THREAD

**Discussion by LiXin-x** [tw93/Pake#101] [General]
地址：[Cubox Web](https://cubox.pro/my)
`https://cubox.pro/my`
Cubox 是一站式信息收集、阅读、管理和回顾，善用网络碎片构建个人知识库的效率软件。
本身有 Mac 版，从 iOS 移植，在桌面使用始终感到变扭。
谢谢开发者。

**Comment by tw93** [comment]
打包了，效果还可以，你试试看
[Cubox_1.0.0_universal.dmg.zip](https://github.com/tw93/Pake/files/10084897/Cubox_1.0.0_universal.dmg.zip)

![image](https://user-images.githubusercontent.com/8736212/203795121-82ec11cb-cfb1-4f24-b722-beb5585a5c99.png)

![image](https://user-images.githubusercontent.com/8736212/203795135-84d000fa-8253-46e0-a07e-8e54217ac062.png)

**Comment by LiXin-x** [comment]
非常感谢🙏，目前体验了一下，效果很不错！

**Comment by LiXin-x** [comment]
我不了解 macOS 开发，我谷歌没找到我想要的答案，也不知道应该搜什么关键字，很好奇是 mac App 的应用安装都是拖拽到「应用程序」，为什么会有以下的区别：

* 你帮我打了个 Cubox 的安装包，我下载安装，自己又打了一个包，两者并不冲突。（不共享应用数据）
* 我自己按照你建议的也尝试去打包，发现安装包名字不同都能安装，但是两者打开是共享数据的。（QQ邮箱同时登录同时退出了）

有空的话，可以简单给我解释一下吗？谢谢
